app/controllers/tags_controller.rb

```diff
@@ -25,17 +25,9 @@ def create
         new_mmt.save
 
         my_module = new_mmt.my_module
-        Activities::CreateActivityService
-          .call(activity_type: :add_task_tag,
-                owner: current_user,
-                subject: my_module,
-                project:
-                  my_module.experiment.project,
-                team: current_team,
-                message_items: {
-                  my_module: my_module.id,
-                  tag: @tag.id
-                })
+
+        log_activity(:create_tag, @tag.project, tag: @tag.id, project: @tag.project.id)
```

Comment:
Shouldn't this be outside `if params.include? "my_module_id"'` block?

Reply:
Fix